src/dokan-components.css

```
@@ -0,0 +1,526 @@
+/* Utility class for primary background */
+.bg-primary-500 {
+    background-color: var(--color-primary);
+
+    &:focus {
+        outline-color: var(--color-primary);
+    }
+}
+
+.dokan-dashboard header.dokan-ai-prompt {
+    margin: 0 0 10px 0;
+    border-bottom: 1px solid #ededed;
```

Stylelint (17.2.0)

[error] Unexpected duplicate "border-bottom" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)